src/Microsoft.VisualStudio.Threading/DispatcherExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿/********************************************************
+*                                                        *
+*   © Copyright (C) Microsoft. All rights reserved.      *
+*                                                        *
+*********************************************************/
+
+#if NET45
```

Comment:
As I said, @sharwell, this is already a pattern we use. This is not the first API in this library to exist on net45 that doesn't exist in the netstandard one. Before we targeted netstandard (or the PCL profiles for that matter), we targeted net45. And some of those APIs are not possible on netstandard because they expose types that are not available on netstandard.

Having facade assemblies is fine (I do that for PCLCrypto) when netstandard has nothing to offer that would allow a library to be implemented. In this case, there is no reason for us to target .NET Core directly, since .netstandard itself can implement 95% of the library and the remaining 5% .NET Core can't offer either. So our netstandard library carries implementation. No one targeting net45+ is targeting netstandard of .netcore anyway so I can't imagine a scenario where folks get runtime failures with this model unless they are mis-applying net45 targeted DLLs onto .NET Core.